### PR TITLE
Feature/snapshot menu

### PR DESCRIPTION
**Pull request by deonjo**
Finally we can delete snapshots! And rename them.

Screenshot 2019-03-21 13 55 56

### CODE REVIEW

File: src/views/arrangement/arrangement.jsx [hunk 1]
```diff
@@ -147,13 +157,17 @@ export class Arrange extends Component {
                 </Grid>
                 <Tabs 
                     active={this.state.activeTab} 
-                    onTabSwitch={this.handleTabSwitch.bind(this)}
-                    onTabAdd={this.handleTabAdd.bind(this)}
+                    onTabSwitch={this.handleTabSwitch}
```
Comment (jeffchiucp) [inline src/views/arrangement/arrangement.jsx]: @deonjo is this refactoring the code for Tab Switch?

File: src/views/arrangement/arrangement.jsx [hunk 2]
```diff
@@ -31,14 +32,23 @@ export class Arrange extends Component {
         this.setState({activeTab: active})
     }
 
-    handleTabAdd() {
+    handleTabAdd = () => {
```
Comment (jeffchiucp) [inline src/views/arrangement/arrangement.jsx]: is this ES6  fat arrow  ?

File: src/views/arrangement/arrangement.jsx [hunk 3]
```diff
@@ -147,13 +157,17 @@ export class Arrange extends Component {
                 </Grid>
                 <Tabs 
                     active={this.state.activeTab} 
-                    onTabSwitch={this.handleTabSwitch.bind(this)}
-                    onTabAdd={this.handleTabAdd.bind(this)}
+                    onTabSwitch={this.handleTabSwitch}
+                    onTabAdd={this.handleTabAdd}
```
Comment (jeffchiucp) [inline src/views/arrangement/arrangement.jsx]: this is the new refactoring handleTabAdd

File: src/views/arrangement/arrangement.jsx (hunk 2 above)
Comment (deonjo) [reply src/views/arrangement/arrangement.jsx]: yes

Comment (jeffchiucp) [reply src/views/arrangement/arrangement.jsx]: Cool. so you keep the context, and it allows you define the functions

Comment (jeffchiucp) [reply src/views/arrangement/arrangement.jsx]: nice!

**Pull request merged**